### ISSUE TITLE
when is the next release to clear npm audit unziper < 0.8.13?

### ISSUE THREAD

**Issue by csantanapr** [ibmdb/node-ibm_db#444]
I'm currently getting flag because ibm_db@2.4.0 depends on `"unzipper": "^0.7.6"`
```

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Arbitrary File Write via Archive Extraction                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ unzipper                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.8.13                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ ibm_db                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ ibm_db > unzipper                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/680                       │
└───────────────┴──────────────────────────────────────────────────────────────┘

```
Can a new patch release like `2.4.1` be publish to npm with the new version of unziper, this is already in master since June https://github.com/ibmdb/node-ibm_db/issues/417

**Comment by hAlavi** [comment]
AS this is not a change in the code and just needed to 'npm publish' the latest master branch, Would you mind please speed up the publish process. Coz most of the security sensitive projects which have pre-commit/push processes will be affected

**Comment by bimalkjha** [comment]
@hAlavi We'll do that soon. The unzipper version is already changed to 0.8.14 in package.json on github. Thanks.

**Comment by Teamop** [comment]
@bimaljha friendly ping. Any news? thanks

**Comment by bimalkjha** [comment]
@hAlavi @Teamop We have published ibm_db@2.4.1 which should address it. Thanks.
